src/test/java/net/minecraftforge/debug/gameplay/loot/GlobalLootModifiersTest.java

```diff
@@ -0,0 +1,193 @@
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.FurnaceRecipe;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraft.util.JSONUtils;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.LootContext;
+import net.minecraft.world.storage.loot.LootParameterSets;
+import net.minecraft.world.storage.loot.LootParameters;
+import net.minecraft.world.storage.loot.LootTable;
+import net.minecraft.world.storage.loot.conditions.ILootCondition;
+import net.minecraftforge.common.ModDimension;
+import net.minecraftforge.common.loot.IGlobalLootModifierSerializer;
+import net.minecraftforge.common.loot.LootModifier;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventBusSubscriber;
+import net.minecraftforge.items.ItemHandlerHelper;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.ObjectHolder;
+
+@Mod(GlobalLootModifiersTest.MODID)
+public class GlobalLootModifiersTest {
+    public static final String MODID = "global_loot_test";
+    public static final boolean ENABLE = true;
+    @ObjectHolder(value = MODID)
+    public static final Enchantment smelt = null;
+    public GlobalLootModifiersTest() { }
+
+    @EventBusSubscriber(modid = MODID, bus = EventBusSubscriber.Bus.MOD)
+    public static class EventHandlers {
+        @SubscribeEvent
+        public static void registerEnchantments(@Nonnull final RegistryEvent.Register<Enchantment> event) {
+            if (ENABLE) {
+                event.getRegistry().register(new SmelterEnchantment(Rarity.UNCOMMON, EnchantmentType.DIGGER, new EquipmentSlotType[] {EquipmentSlotType.MAINHAND}).setRegistryName(new ResourceLocation(MODID,"smelt")));
+            }
+        }
+
+        @SubscribeEvent
+        public static void registerModifierSerializers(@Nonnull final RegistryEvent.Register<IGlobalLootModifierSerializer<?>> event) {
+            if (ENABLE) {
+                event.getRegistry().register(
+                		new WheatSeedsConverterModifier.Serializer().setRegistryName(new ResourceLocation(MODID,"wheat_harvest"))
+                );
+                event.getRegistry().register(new SmeltingEnchantmentModifier.Serializer().setRegistryName(new ResourceLocation(MODID,"smelting")));
+                event.getRegistry().register(new SilkTouchTestModifier.Serializer().setRegistryName(new ResourceLocation(MODID,"silk_touch_bamboo")));
+            }
+        }
+    }
+
+    private static class SmelterEnchantment extends Enchantment {
+        protected SmelterEnchantment(Rarity rarityIn, EnchantmentType typeIn, EquipmentSlotType[] slots) {
+            super(rarityIn, typeIn, slots);
+        }
+    }
+
+    /**
+     * The smelting enchantment causes this modifier to be invoked, via the smelting loot_modifier json
+     * @author Draco18s
```

Comment:
no

Reply:
No *what?* "Don't put javadoc" "don't put @author"? What am I doing wrong?

Reply:
I did ti specifically on the author line, so it's about the @author, we don't do that in Forge.
Javadocs are good, Ego stroking is bad. {Not saying that's what you're doing but you get enough credit in the git log it doesn't need to be in the source}

Reply:
The comment widget includes four lines of code and doesn't indicate exactly which line you pinged. Removing @author, got it. Thanks. I didn't put it to stoke my ego, it was added automatically when I inserted the javadoc.

Reply:
Just for future reference, the widget will always show on the line that it was commented on, the 3 lines above it are for context.

Reply:
Fair enough. Just with the Github UI was a little better, I guess. Anyway, removing those lines is easy enough.